utils.go

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package virtcontainers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+func fileCopy(srcPath, dstPath string) error {
```

Comment:
A thought occurred - extended attributes and security contexts. This doesn't appear to be performance-critical code so it would honestly be easier and safer (and avoids reinventing the wheel ;) to call out to `cp` (`exec.LookPath()`, *etc*) as that will handle such complexities for us. 

Mr Pike would seem to agree: https://go-review.googlesource.com/c/1591/9/src/io/ioutil/ioutil.go#77.